features/fencedframe.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-21
-name: Fenced Frame
-description: TODO
+name: <fencedframe>
+description: The `<fencedframe>` element embeds content in a more secure and privacy-preserving way compared to `<iframe>` elements.
```

I think it'd be nice if we made this parallel to iframes themselves:

```suggestion
description: The `<fencedframe>` element embeds another HTML page into the current page in a more secure and privacy-preserving way than `<iframe>` elements.
```